ops/docker-compose.yml

```diff
@@ -200,6 +200,41 @@ services:
       ENABLE_GAS_REPORT: 1
       NO_NETWORK: 1
 
+  teleportr_postgres:
```

Comment:
Are you planning on adding integration tests for teleportr? Eventually the docker compose setup will become unwieldy for local development using too many resources and we don't have a good way to define sets of integration tests that bring up parts of the stack to specifically test. I think adding teleportr is fine now fwiw

Reply:
Oh I see that `releportr` has `replicas: 0` but the postres does not

Reply:
good catch, yes i'm planning to add itests but will do so in a follow up. this pr will just have the bulk of the teleportr service